documentation/misc/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,103 @@
+> Release Issue Template
+
+# Lotus X.Y.Z Release
+
+We're happy to announce Lotus X.Y.Z...
+
+## 🗺 What's left for release
+
+<List of items with PRs and/or Issues to be considered for this release>
+
+## 🚢 Estimated shipping date
+
+<Date this release will ship on if everything goes to plan (week beginning...)>
+
+## 🔦 Highlights
+
+< top highlights for this release notes >
+
+## Changelog
+
+< changelog generated by bin/mkreleaselog >
+
+## ✅ Release Checklist
+
+First steps:
+
+  - [ ] Fork a new branch (`release/vX.Y.Z`) from `master` and make any further release related changes to this branch. If any "non-trivial" changes get added to the release, uncheck all the checkboxes and return to this stage.
+  - [ ] Prep the changelog
+  - [ ] Bump the version in `version.go` in the `master` branch to `vX.(Y+1).0-dev`.
+  - [ ] Follow the RC release process to cut the first RC.
+    
+Prepping an RC:
+
+- [ ] version string in `build/version.go` has been updated (in the `release/vX.Y.Z` branch).
+- [ ] tag commit with `vX.Y.Z-rcN`
+- [ ] cut a pre-release on [github](https://github.com/filecoin-project/lotus/releases)
+
+Testing an RC:
+
+- [ ] **Stage 0 - Automated Testing**
+  - Automated Testing
+    - [ ] CI: Ensure that all tests are passing.
+    - [ ] Testground tests
+
+- [ ] **Stage 1 - Internal Testing**
+  - Upgrade our testnet infra
+    - [ ] 1 bootstrap node
+    - [ ] 1 miner
+    - [ ] Scratch nodes
+    - [ ] Wait 24 hours
+    - [ ] Remaining testnet infra
+  -  Upgrade our mainnet infra
+    - [ ] Subset of development full archival nodes
+    - [ ]Subset of bootstrappers (1 per region)
+    - Report on new block validation time
+    - TODO: What other stats would we care about?
+    - If anything has worsened significantly, investigate + fix
+  - Confirm the following work (some combination of Testground / Calibnet / Mainnet / MinerX)
```

Comment:
```suggestion
  - Confirm the following work (some combination of Testground / Calibnet / Mainnet / lotus early testers)
```
```suggestion
  - Confirm the following work (some combination of Testground / Calibnet / Mainnet / MinerX)
```
```suggestion
  - Confirm the following work (some combination of Testground / Calibnet / Mainnet / MinerX)
```

Reply:
Let's call them "Beta Users" rather than "lotus early testers" or "MinerX".  The system has gone through some testing by the time these communities are engaged.